Makefile

```diff
@@ -35,7 +35,12 @@ table.ext: .pre-build
 	go run cmd/make/make.go -targets=table-extension -linkstamp
 
 osqueryi-tables: table.ext
-	osqueryd -S --extension ./build/darwin/tables.ext
+	osqueryd -S --allow-unsafe --extension ./build/darwin/tables.ext
+sudo-osqueryi-tables: table.ext
```

Comment:
Is there a reason to prefer this over `sudo make osqueryi-tables`?

Reply:
That would trigger some weird build and mod caching as root, likely leaving some random files on disk not owned by the user.

I am, generally speaking, against running commands like `make` under sudo, they have a lot unexpected effect. 

I'm not totally sure we need these targets. They're here primarily for convinience